test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/npd-binding.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the node-problem-detector
+# which is used to get and patch the node health status.
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: node-problem-detector-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
@wojtek-t Yes, I can create these roles. The question however is if these roles should be created in kubemark/resources/..../rbac-roles or they should be part of the bootstrapped roles. If indeed the role (and maybe the binding as well?) are created as part of the bootstrapping, we wouldn't have to add much in the kubemark code. What do you think about this @deads2k ?
FYI, issue https://github.com/kubernetes/kubernetes/issues/40021 is now tracking this problem.